packages/libs/deer/macros/rust-toolchain.toml

```diff
@@ -1,2 +0,0 @@
-[toolchain]
-channel = "nightly-2022-09-27"
```

Comment:
Probably a good call to remove this as it's desynced with the parent create (which is at 
`nightly-2022-11-14`)

Reply:
cc: @indietyp beware that we are now using the parent toolchain for the macro folder; likely not a problem though